pkl-core/src/test/files/LanguageSnippetTests/input/syntax/shebang.pkl

```diff
@@ -0,0 +1,2 @@
+#!/usr/bin/env pkl eval
+foo = 1
```

Comment:
I wonder if we should allow this? Your implementation means that we do:

```

#!/usr/bin/env pkl eval

foo = 1
```

*nix will only look at the _start_ of a file for the shebang line, so trying to run this file as an executable this wouldn't work.